### DISCUSSION TITLE
How do i livestream in superview?

### DISCUSSION THREAD

**Discussion by ph140** [gopro/labs#1072] [Q&A] [answered]
Hi, i have been tryin to use QR codes for livestreaming, but it automatically chooses wide, when i want superview, how can I achieve superview?

**Comment by dnewman-gpsw** [comment]
Does the camera support livestreaming in Superview?  There might not be a QR code, if the camera stock firmware doesn't offer that lens for LS.

**Reply by ph140** [reply]
yes, when i use the gopro quik app i get the option of choosing superview when livestreaming. Its a gopro 11. I dont know where i can create custom qr codes. I just used this side, but i dont get any option of choosing lens there

**Reply by dnewman-gpsw** [reply, marked as answer]
Currently there is no QR Code for selecting the digital lens in a Livestream.

**Reply by ph140** [reply]
I see, thanks for the answer:) Just want to say it would be great if there was, since you get that feature when using the app, but not currently in gopro labs qr code
